devel/ixion/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 PortGroup           gitlab 1.0
 
 gitlab.setup        ixion ixion 0.16.1
+revision            1
```

Comment:
In general, adding build dependencies does not require a `revision` increase. Unless it somehow changes the installed files for those who already have it installed.